### PR TITLE
docs: add context to hooks auth snippets

### PR DESCRIPTION
**Pull request by jonas-jonas**
The current documentation for auth in kratos' web hooks lacks context and is difficult to understand ([example](https://ory-community.slack.com/archives/C012RJ2MQ1H/p1660048260538329)) if you don't know what to look for. This PR adds some context around the snippets and (hopefully) makes this topic a bit clearer.

Also, I removed some stray " (quotation mark) from the path in code snippets on the page.

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation within the code base (if appropriate).

### CODE REVIEW

**Review by kmherrmann — DISMISSED** [review]
Thanks for doing this!

File: docs/kratos/self-service/hooks.mdx [hunk 1]
```diff
@@ -202,7 +214,7 @@ file.
 
 ### Before
 
-```yaml title="path/to/my/kratos.config.yml"
+```yaml title=path/to/my/kratos.config.yml"
```
Comment (tomekpapiernik) [inline docs/kratos/self-service/hooks.mdx:217]:
```suggestion
```yaml title=path/to/my/kratos.config.yml
```

File: docs/kratos/self-service/hooks.mdx [hunk 2]
```diff
@@ -214,7 +226,7 @@ selfservice:
 
 ### After
 
-```yaml title="path/to/my/kratos.config.yml"
+```yaml title=path/to/my/kratos.config.yml"
```
Comment (tomekpapiernik) [inline docs/kratos/self-service/hooks.mdx:229]:
```suggestion
```yaml title=path/to/my/kratos.config.yml
```

File: docs/kratos/self-service/hooks.mdx [hunk 3]
```diff
@@ -230,7 +242,7 @@ selfservice:
 The `revoke_active_sessions` will delete all active sessions for that user on successful login. This hook is available for all
 supported methods:
 
-```yaml title="path/to/my/kratos.config.yml"
+```yaml title=path/to/my/kratos.config.yml"
```
Comment (tomekpapiernik) [inline docs/kratos/self-service/hooks.mdx:245]:
```suggestion
```yaml title=path/to/my/kratos.config.yml
```

File: docs/kratos/self-service/hooks.mdx [hunk 4]
```diff
@@ -246,7 +258,7 @@ selfservice:
 The `require_verified_address` will ensure, the user can login only if its email address is verified. This hook is available for
 the `password` method only:
 
-```yaml title="path/to/my/kratos.config.yml"
+```yaml title=path/to/my/kratos.config.yml"
```
Comment (tomekpapiernik) [inline docs/kratos/self-service/hooks.mdx:261]:
```suggestion
```yaml title=path/to/my/kratos.config.yml
```

File: docs/kratos/self-service/hooks.mdx [hunk 5]
```diff
@@ -269,7 +281,7 @@ configuration file.
 
 ### Before
 
-```yaml title="path/to/my/kratos.config.yml"
+```yaml title=path/to/my/kratos.config.yml"
```
Comment (tomekpapiernik) [inline docs/kratos/self-service/hooks.mdx:284]:
```suggestion
```yaml title=path/to/my/kratos.config.yml
```

File: docs/kratos/self-service/hooks.mdx [hunk 6]
```diff
@@ -281,7 +293,7 @@ selfservice:
 
 ### After
 
-```yaml title="path/to/my/kratos.config.yml"
+```yaml title=path/to/my/kratos.config.yml"
```
Comment (tomekpapiernik) [inline docs/kratos/self-service/hooks.mdx:296]:
```suggestion
```yaml title=path/to/my/kratos.config.yml
```

File: docs/kratos/self-service/hooks.mdx [hunk 7]
```diff
@@ -310,7 +322,7 @@ existing and non-existing accounts by checking if `Set-Cookie` was sent as part
 To use this hook, you must first define one or more (for secret rotation) secrets. You can either choose to use the "default"
 secrets or the more specific "cookie" secrets. The other required config is setting the hook in `after` work flows:
 
-```yaml title="path/to/my/kratos.config.yml"
+```yaml title=path/to/my/kratos.config.yml"
```
Comment (tomekpapiernik) [inline docs/kratos/self-service/hooks.mdx:325]:
```suggestion
```yaml title=path/to/my/kratos.config.yml
```

File: docs/kratos/self-service/hooks.mdx [hunk 8]
```diff
@@ -373,7 +385,7 @@ configuration file.
 
 ### After
 
-```yaml title="path/to/my/kratos.config.yml"
+```yaml title=path/to/my/kratos.config.yml"
```
Comment (tomekpapiernik) [inline docs/kratos/self-service/hooks.mdx:388]:
```suggestion
```yaml title=path/to/my/kratos.config.yml
```

**Review by tomekpapiernik — CHANGES_REQUESTED** [review]
You must either leave both of the `"` in `title=` or remove both of them ;)

**Comment by jonas-jonas** [comment]
@tomekpapiernik right, apparently the quotes were only missing on the first few snippets. I added them around all of them. It should be good now.

**Review by tomekpapiernik — APPROVED** [review]

**Pull request merged**